Tests/SwiftLintFrameworkTests/AutomaticRuleTests.generated.swift

```diff
@@ -887,12 +887,6 @@ class VerticalWhitespaceBetweenCasesRuleTests: XCTestCase {
     }
 }
 
-class VerticalWhitespaceClosingBracesRuleTests: XCTestCase {
```

Comment:
Let's keep the auto-generated tests. You can adjust which examples apply to which configurations by specifying the `configuration` parameter in the [`Example` initializer](https://realm.github.io/SwiftLint/Structs/Example.html#/s:18SwiftLintFramework7ExampleV_13configuration20testMultiByteOffsets0F17WrappingInComment0fjK6String0F14DisableCommand0F7OnLinux4file4line24excludeFromDocumentationACSS_ypSgS5bs06StaticM0VSuSbtcfc).

Reply:
Done, though I'm not sure if it was done correctly.

Reply:
You can add the examples in the dedicated test class together with their configuration to the list of default examples. That way, they are automatically tested using the attached configuration.

Reply:
Done.
Assuming the "dedicated test class" you mentioned was in `VerticalWhitespaceClosingBracesRuleTests.swift`, and "the list of default examples" is in `VerticalWhitespaceClosingBracesRuleExamples.swift`.